### PR TITLE
add license of webicons, cc #5500

### PR DESCRIPTION
**Pull request by maruilian11**
Reference:
https://github.com/adamfairhead/webicons/issues/2#issuecomment-6919938

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Let's focus on other old request issues, license is not that important and the new interns will take care of it.

**Comment by maruilian11** [comment]
ok, so I just close this?

File: ajax/libs/webicons/package.json [hunk 1]
```diff
@@ -11,5 +11,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/adamfairhead/webicons.git"
-  }
+  },
+  "license": "CC-BY-3.0"
```
Comment (PeterDaveHello) [inline ajax/libs/webicons/package.json]: What about the US part?

File: ajax/libs/webicons/package.json [hunk 2]
```diff
@@ -11,5 +11,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/adamfairhead/webicons.git"
-  }
+  },
+  "license": "CC-BY-3.0-US"
```
Comment (maruilian11) [inline ajax/libs/webicons/package.json]: @PeterDaveHello is this format ok?

Comment (PeterDaveHello) [inline ajax/libs/webicons/package.json]: Do you see this string in the license list?

Comment (maruilian11) [inline ajax/libs/webicons/package.json]: I only find `CC-BY-3.0` in following license list
https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L63
https://spdx.org/licenses/CC-BY-3.0.html

so I need to separate the license to `CC-BY-3.0` and US part?

Comment (PeterDaveHello) [inline ajax/libs/webicons/package.json]: You need to confirm what's SPDX license id of it ...

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by maruilian11** [comment]
@PeterDaveHello still get no response from the author.

**Comment by sufuf3** [comment]
@maruilian11 as https://freedomdefined.org/Licenses/CC-BY-3.0 said the https://creativecommons.org/licenses/by/3.0/us/ is for the legal code of the license.
I think many countries use CC-By-3.0 for their legal code of the license, such as Taiwan https://creativecommons.org/licenses/by/3.0/tw/

So I think we can use `CC-BY-3.0` as the license, what do you think?
Thank you.

**Comment by maruilian11** [comment]
@sufuf3 I tried before. Because `CC-BY-3.0-US` doesn't exist in SPDX licenses IDs, we need to confirm  it with the author.
@PeterDaveHello Since the author hasn't replied for a long time, can we add it with `CC-BY-3.0` first?

**Comment by PeterDaveHello** [comment]
I don't believe that we can use `CC-BY-3.0` "because" the author didn't reply, can you please confirm if it make sense to use `CC-BY-3.0` represent the US version first?

**Comment by maruilian11** [comment]
@PeterDaveHello 
https://creativecommons.org/licenses/by/3.0/us/
https://creativecommons.org/licenses/by/3.0/tw/
Seems like they just write in different language to explain the license, so I think we can use `CC-BY-3.0`.

**Comment by PeterDaveHello** [comment]
I don't think they are the same, see the reference:
http://lucien.cc/20150927-3-0-%E5%8F%B0%E7%81%A3%E9%82%84%E6%98%AF-4-0-%E5%9C%8B%E9%9A%9B-%E4%B8%8D%E5%90%8C%E7%89%88%E6%9C%AC%E7%9A%84%E5%89%B5%E7%94%A8cc-%E6%8E%88%E6%AC%8A%E6%A2%9D%E6%AC%BE%E8%A9%B2%E5%A6%82/

**Comment by PeterDaveHello** [comment]
May need more discussions and suggestions, or even asking people good at licensing.

**Comment by sashberd** [comment]
@maruilian11 @PeterDaveHello 
I read this [one ](https://creativecommons.org/licenses/by/3.0/us/deed.ru)of global and this [one ](https://creativecommons.org/licenses/by/3.0/deed.ru)of USA on my native language. The difference is only covering area of license. that is  all.
Suggest to use CC-BY-3.0 (global)
Another way: Why we cannot add another library to our license list?

File: ajax/libs/webicons/package.json (hunk 2 above)
Comment (MattIPv4) [reply ajax/libs/webicons/package.json]: This is a valid CC license, just not in the SPDX list.

**Pull request merged**